schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+ 	id INT NOT NULL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BIT,
```

Comment:
- [ ] neutered should have a boolean datatype BIT datatype stores **1 or 0 or Null**. Kindly change **BIT** to **boolean** :blush:

Reply:
thank you @HAMMAS-SALEEM. Changes has been made